### PR TITLE
ChatGLM3, Baichuan2 and Qwen1.5 QLoRA example

### PR DESCRIPTION
**Pull request by qiyuangong**
## Description

QLora alpaca example for Baichuan2 and Qwen1.5 

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

File: python/llm/example/GPU/LLM-Finetuning/QLoRA/alpaca-qlora/qlora_finetune_baichuan2_7b_arc_1_card.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# You could also specify `--base_model` to the local path of the huggingface model checkpoint folder and `--data_path` to the local path of the dataset JSON file
+python ./alpaca_qlora_finetuning.py \
+    --base_model "meta-llama/Baichuan2-7B-Chat" \
+    --data_path "yahma/alpaca-cleaned" \
+    --output_dir "./ipex-llm-qlora-alpaca" \
+    --
```
Comment (plusbang) [inline python/llm/example/GPU/LLM-Finetuning/QLoRA/alpaca-qlora/qlora_finetune_baichuan2_7b_arc_1_card.sh:22]: additional `--` ?

File: python/llm/example/GPU/LLM-Finetuning/QLoRA/alpaca-qlora/qlora_finetune_qwen15_7b_arc_1_card.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# You could also specify `--base_model` to the local path of the huggingface model checkpoint folder and `--data_path` to the local path of the dataset JSON file
+python ./alpaca_qlora_finetuning.py \
+    --base_model "meta-llama/Qwen1.5-7B-Chat" \
+    --data_path "yahma/alpaca-cleaned" \
+    --output_dir "./ipex-llm-qlora-alpaca" \
```
Comment (plusbang) [inline python/llm/example/GPU/LLM-Finetuning/QLoRA/alpaca-qlora/qlora_finetune_qwen15_7b_arc_1_card.sh:21]: I suppose there don't need more parameter ? maybe remove the last `\`.

**Review by plusbang — APPROVED** [review]
Other LGTM

File: python/llm/example/GPU/LLM-Finetuning/QLoRA/alpaca-qlora/qlora_finetune_baichuan2_7b_arc_1_card.sh (hunk 1 above)
Comment (qiyuangong) [reply python/llm/example/GPU/LLM-Finetuning/QLoRA/alpaca-qlora/qlora_finetune_baichuan2_7b_arc_1_card.sh]: Removed

**Comment by qiyuangong** [comment]
In latest transformers, setting tokenizer is no more necessary. Wrose more, it will cause problems when config is not found in some models. In this pr, we removed padding and padding_side to avoid errors.

**Pull request merged**